maps/randomvaults/oldarmory.dmm

```diff
@@ -0,0 +1,165 @@
+"bS" = (/obj/structure/rack,/obj/map/spawner/space/weapons2,/obj/map/spawner/space/weapons2,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bT" = (/obj/structure/rack,/obj/map/spawner/security/armor,/obj/map/spawner/security/armor,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bU" = (/obj/structure/rack,/obj/map/spawner/security/weapons,/obj/map/spawner/security/weapons,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bV" = (/obj/structure/rack,/obj/item/device/batterer,/obj/item/weapon/gun/grenadelauncher,/obj/item/weapon/storage/box/emps,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bW" = (/obj/structure/bed/chair{dir = 4},/mob/living/simple_animal/hostile/humanoid/syndicate/ranged/space,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bX" = (/obj/structure/table/reinforced{icon_state = "table"},/obj/item/trash/syndi_cakes,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bY" = (/obj/structure/table/reinforced{icon_state = "table"},/obj/item/weapon/pen/sleepypen,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"bZ" = (/obj/structure/bed/chair{dir = 8},/mob/living/simple_animal/hostile/humanoid/syndicate/ranged/space,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"ca" = (/obj/machinery/door/airlock{name = "Disguises"},/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cb" = (/obj/structure/closet/gimmick/tacticool,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cc" = (/obj/structure/closet,/obj/item/clothing/head/helmet/volnutthelmet,/obj/item/clothing/suit/armor/volnutt,/obj/item/weapon/gun/energy/mmlbuster,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cd" = (/obj/machinery/porta_turret{emagged = 1},/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"ce" = (/obj/structure/sign/securearea,/turf/simulated/wall/r_wall,/area/vault/oldarmory)
+"cf" = (/obj/structure/painting/random,/turf/simulated/wall/r_wall,/area/vault/oldarmory)
+"cg" = (/obj/machinery/door/airlock/command{name = "Computer Chamber"},/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"ch" = (/obj/machinery/vending/wallmed1,/turf/simulated/wall/r_wall,/area/vault/oldarmory)
+"ci" = (/obj/machinery/vending/suitdispenser,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cj" = (/obj/structure/safe,/obj/item/toy/gasha/newcop,/obj/item/toy/syndicateballoon,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"ck" = (/obj/machinery/door/airlock/vault{locked = 1; name = "Communications Relay"},/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cl" = (/obj/machinery/vending/shoedispenser,/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
+"cm" = (/obj/machinery/syndicate_beacon{desc = "This seems to be what the Syndicate agents here are using to communicate with their leaders. Maybe they'd be willing to hire you?"; name = "Syndicate Communications Relay"},/turf/simulated/shuttle/floor4,/area/vault/oldarmory)
```

Comment:
It's a very rare Xenoarchaeology find. If you're not an antagonist, it's a 50% chance of becoming a traitor with two objectives, with the first one being something more unusual than the normal objectives like "cut power to 80% or more of the station". (The other half is the beacon blowing up in your face.) Yes, it provides you with an uplink as well. 

If you're already an antagonist, it doesn't do shit.